bbc/simorgh
# WSTEAM1-847: Live Page - Image Caption Spacing

Resolves JIRA [WSTEAM1-847](https://jira.dev.bbc.co.uk/browse/WSTEAM1-847)

Overall changes
======
Updates padding values to fix spacing underneath images & videos on Live Pages.

Code changes
======
- Correct padding values applied to Images & Videos in Posts on the Live Page (16px in designs).
- Add a default bottom padding to the mediaLoader. (maybe out of scope out of the original ticket, but fixes a similiar bug that I noticed on the development env).

Testing
======
- Checkout this branch `WSTEAM1-847-figure-spacing`
- Run the NextJs app locally and visit http://localhost:7081/burmese/live/c51v5pq2d2wt?renderer_env=live
- Check there is correct spacing underneath imageWithCaption (e.g. the one captioned 'နေပြည်တော်က ၂၀၂၃ ခုနှစ်၊ လွတ်လပ်ရေးနေ့စစ်ရေးပြပွဲ')
- Check there is correct spacing underneath image (post titled 'ရခိုင် ၆ မနယ်မှာ စစ်ကောင်စီနဲ့ ရက္ခိုင့်တပ်တော်တို့ တိုက်ပွဲပြင်းထန်နေ')
- Check there is correct spacing underneath video (Captioned 'မင်းပြားမကို AA ထိန်းချုပ်')

There are visuals attached to the ticket:  [WSTEAM1-847](https://jira.dev.bbc.co.uk/browse/WSTEAM1-847)
Can be compared to the live article: https://www.bbc.com/burmese/live/c51v5pq2d2wt (note this will have the old version of the media player)
Can be compared to Latest running locally (this will have the new version of the media player)

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

- UX designs: https://www.figma.com/file/kBQdRE6Tc5XZ9L2HejAUgD/Live-mvp---post-stream-%26-pagination---handover?type=design&node-id=502-60709&mode=design&t=tFv274qJMU2DtNrv-4

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
